packages/dso-toolkit/src/components/card/card.args.ts

```diff
@@ -6,20 +6,17 @@ import { AnchorArgs } from "../anchor/anchor.args.js";
 import { Button } from "../button/button.models.js";
 import { Label } from "../label/label.models.js";
 import { Toggletip } from "../toggletip/toggletip.models.js";
-import { Card, imageShapes } from "./card.models.js";
+import { Card } from "./card.models.js";
 import { SlideToggle } from "../slide-toggle";
 
 export interface CardArgs {
   label: string;
-  href?: string;
+  href: string;
+  active: boolean;
   mode?: AnchorArgs["mode"];
   selectable: boolean;
   interactions: Array<Button | Label | Toggletip<never> | SlideToggle>;
-  image: string | undefined;
-  imageAlt: string | undefined;
-  imageShape: (typeof imageShapes)[number];
-  clickable: boolean;
-  dsoCardClicked?: HandlerFunction;
+  dsoCardClick?: HandlerFunction;
```

Comment:
Deze wordt altijd door Storybook gemaakt, toch? Ofwel, deze property is niet optioneel?

Reply:
Verplicht gemaakt.